hack/krmFunctionBenchmark/readme.md

```diff
@@ -0,0 +1,28 @@
+# Benchmark for KRM functions in Kustomize
+
+## Pre-request
+
+ - You have to use Kustomize version with KRM function supported.
+ - You need to have Docker running on your machine.
```

Comment:
- [kustomize v3.7.0 or higher](https://kubernetes-sigs.github.io/kustomize/installation) (support for KRM Config Functions).
 - [docker](docker.com)